tests/test_Units.py

```diff
@@ -59,3 +61,50 @@ def test_units(xmlfile, csc, topic):
             "Unit '" + unit.text + "' in " + str(
                 xmlfile.name
             ) + " does not meet astropy standards."
+
+
+@pytest.mark.parametrize("xmlfile,csc,topic", ts_xml.get_xmlfile_csc_topic())
+def test_string_units(xmlfile, csc, topic):
+    """Test that the <Units> field for STRING-type attributes is 'unitless.'
+
+    Parameters
+    ----------
+    xmlfile : `pathlib.Path`
+        Full filepath to the Commands or Events XML file for the CSC.
+    csc : `testutils.subsystems`
+        Name of the CSC
+    topic : `xmlfile.stem`
+        One of ['Commands','Events','Telemetry']
+    """
+    saltype = "SAL" + topic.rstrip("s")
+    # Check for known issues.
+    jira = check_for_issues(csc, topic)
+    if jira:
+        pytest.skip(
+            jira
+            + ": "
+            + str(xmlfile.name)
+            + " <Unit> fields do not conform to astropy standards."
```

Comment:
Please consider an f string for readability. The following should do it (the str is implied by the {}):
```
f"{jira}: {xmlflie.name} <Unit> fields do not conform to astropy standards."
```

Reply:
Fixed